README.adoc

```diff
@@ -14,26 +14,27 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== For All Your Social Media Needs
+
+Media Socializer aims to provide you with one-stop access to your favourite social media platforms.
+It keeps track of the various platforms your friends are using and sends you straight to their page if you so desire.
+You can also store them as contacts in case you need to communicate with them offline.
+Check out the social feeds of different people on multiple platforms all in a single window!
+
+*Go on, grab the app and get started now!*
 
 == Site Map
 
 * <<UserGuide#, User Guide>>
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
```

Comment:
Could you delete `docs/LearningOutcomes.adoc` as well?

Reply:
OK. I've removed the document in a new commit.